### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by wilwade**
- Roadmap Updates
- Remove disclaimers
- Bump to v1.0.0

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -86,13 +83,98 @@ Here is a high level roadmap to what content is coming next!
...
+
+### Future
+
+Below is a tentative roadmap to what protocol features are being developed next!
+Order is approximately in current priority order.
+
+#### Chain Migration
+*Status: In progress*
+
+Ethereum's cost is just too high.
```
Comment (enddynayn) [inline pages/index.md:96]: this should be all caps with an exclamation at the end.

**Review by enddynayn — APPROVED** [review]

**Review by rosalinekarr — APPROVED** [review]

**Review by shannonwells — APPROVED** [review]
Nothing to complain about here

**Review by aneyzberg — APPROVED** [review]
lgtm!

**Review by acruikshank — APPROVED** [review]
Looks good.

**Pull request merged**